cmd/cloud-controller-manager/nodeipamcontroller.go

```diff
@@ -98,14 +98,14 @@ func startNodeIpamController(ccmConfig *cloudcontrollerconfig.CompletedConfig, n
 	if len(strings.TrimSpace(nodeIPAMConfig.ServiceCIDR)) != 0 {
 		_, serviceCIDR, err = net.ParseCIDR(nodeIPAMConfig.ServiceCIDR)
 		if err != nil {
-			klog.Warningf("Unsuccessful parsing of service CIDR %v: %v", nodeIPAMConfig.ServiceCIDR, err)
+			klog.InfoS("Unsuccessful parsing of service CIDR", "CIDR", nodeIPAMConfig.ServiceCIDR, "err", err)
```

Comment:
Warningf needs to be mapped to either InfoS or ErrorS. Given that this is an error in configuration at startup which is going to hamper the controller doing its work, I think it would be more appropriate to use ErrorS here.